test/RadworksGovernor.t.sol

```diff
@@ -137,10 +138,589 @@ abstract contract Propose is ProposalTest {
+    uint256 _amountToken,
+    address _receiver,
+    uint256 _seed
+  ) public {
+    IERC20 _token = _randomERC20Token(_seed);
+    _assumeReceiver(_receiver);
+
+    vm.deal(TIMELOCK, _amountETH);
+    uint256 _timelockETHBalance = TIMELOCK.balance;
+    uint256 _receiverETHBalance = _receiver.balance;
+
+    // Bound _amountToken by the number of tokens the timelock currently controls.
+    uint256 _timelockTokenBalance = _token.balanceOf(TIMELOCK);
+    uint256 _receiverTokenBalance = _token.balanceOf(_receiver);
+    _amountToken = bound(_amountToken, 0, _timelockTokenBalance);
+
+    _upgradeToBravoGovernor();
+
+    // Craft a new proposal to send ETH and tokens.
+    address[] memory _targets = new address[](2);
+    uint256[] memory _values = new uint256[](2);
+    bytes[] memory _calldatas = new bytes[](2);
+
+    // First call transfers tokens.
+    _targets[0] = address(_token);
+    _calldatas[0] =
+      _buildProposalData("transfer(address,uint256)", abi.encode(_receiver, _amountToken));
+
+    // Second call sends ETH.
+    _targets[1] = _receiver;
+    _values[1] = _amountETH;
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets,
+      _values,
+      _calldatas,
+      "Transfer tokens and ETH via the new Governor",
+      FOR // Vote/suppport type.
+    );
+
+    // Ensure the ETH was transferred.
+    assertEq(_receiver.balance, _receiverETHBalance + _amountETH);
+    assertEq(TIMELOCK.balance, _timelockETHBalance - _amountETH);
+
+    // Ensure the tokens were transferred.
+    assertEq(_token.balanceOf(_receiver), _receiverTokenBalance + _amountToken);
+    assertEq(_token.balanceOf(TIMELOCK), _timelockTokenBalance - _amountToken);
+  }
+
+  function testFuzz_NewGovernorFailedProposalsCantSendETH(uint256 _amount, address _receiver)
+    public
+  {
+    _assumeReceiver(_receiver);
+    vm.deal(TIMELOCK, _amount);
+    uint256 _timelockETHBalance = TIMELOCK.balance;
+    uint256 _receiverETHBalance = _receiver.balance;
+
+    _upgradeToBravoGovernor();
+
+    // Craft a new proposal to send ETH.
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    _targets[0] = _receiver;
+    _values[0] = _amount;
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets,
+      _values,
+      new bytes[](1), // There is no calldata for a plain ETH call.
+      "Transfer some ETH via the new Governor",
+      (_amount % 2 == 1 ? AGAINST : ABSTAIN) // Randomize vote type.
+    );
+
+    // Ensure ETH was *not* transferred.
+    assertEq(_receiver.balance, _receiverETHBalance);
+    assertEq(TIMELOCK.balance, _timelockETHBalance);
+  }
+
+  function testFuzz_NewGovernorCanUpdateSettingsViaSuccessfulProposal(
+    uint256 _newDelay,
+    uint256 _newVotingPeriod,
+    uint256 _newProposalThreshold
+  ) public {
+    // The upper bounds are arbitrary here.
+    _newDelay = bound(_newDelay, 0, 50_000); // about a week at 1 block per 12s
```

Comment:
nit: may be overkill but perhaps the following assertion could be useful: that the current delay, voting period, prop threshold does not equal corresponding current value

Reply:
Done: 3a921d8b5058860c8dd423fd85a932cf7861160b